source/extensions/filters/http/aws_lambda/config.cc

```diff
@@ -48,7 +48,8 @@ Http::FilterFactoryCb AwsLambdaFilterFactory::createFilterFactoryFromProtoTyped(
   const std::string region = arn->region();
   auto signer = std::make_shared<Extensions::Common::Aws::SignerImpl>(
       service_name, region, std::move(credentials_provider),
-      context.mainThreadDispatcher().timeSource());
+      context.mainThreadDispatcher().timeSource(),
+      Extensions::Common::Aws::AwsSigV4HeaderExclusionVector{});
```

Comment:
Should there be a TODO here for allow configuration of this? Wouldn't the same problem exist for this filter?

Reply:
Yes, I think both the lambda and grpc iam filters could use the same change, otherwise they'd only be able to use the existing default header exclusion (and the spirit of this PR is to avoid adding more such headers). I'll add a TODO.

Reply:
Yeah, I think we'd eventually want this for the lambda and grpc iam filters. I'll add a TODO.